code/modules/projectiles/projectile/special_projectiles.dm

```diff
@@ -366,3 +366,109 @@
 	var/mob/living/simple_animal/hostile/mimic/copy/ranged/R = new /mob/living/simple_animal/hostile/mimic/copy/ranged(T, G, firer)
 	if(ismob(target))
 		R.target = target
+
+/obj/effect/ebeam/curse_arm
+	name = "curse arm"
+
+/obj/item/projectile/curse_hand
+	name = "curse hand"
+	icon_state = "cursehand0"
+	base_icon_state = "cursehand"
+	hitsound = 'sound/effects/curse/curse4.ogg'
+	layer = LARGE_MOB_LAYER
+	damage = 0
+	speed = 0.5
+	range = 16
+	forcedodge = -1
+	var/burn_damage = 10
+	var/weaken_damage = 2 SECONDS
+	var/datum/beam/arm
+	var/handedness = 0
+
+/obj/item/projectile/curse_hand/Initialize(mapload)
+	. = ..()
+	handedness = prob(50)
+	icon_state = "[base_icon_state][handedness]"
+
+/obj/item/projectile/curse_hand/Destroy()
+	QDEL_NULL(arm)
+	return ..()
+
+/obj/item/projectile/curse_hand/update_icon_state()
+	icon_state = "[base_icon_state]0[handedness]"
+	return ..()
+
+/obj/item/projectile/curse_hand/fire(setAngle)
+	if(QDELETED(src)) //I'm going to try returning nothing because if it's being deleted, surely we don't want anything to happen?
+		return
+	if(starting)
+		arm = Beam(starting, icon_state = "curse[handedness]", beam_type=/obj/effect/ebeam/curse_arm, maxdistance = 20)
+	..()
+
+/obj/item/projectile/curse_hand/on_range()
+	finale() //qwertodo low priority: this isn't working right?
+	. = ..()
```

Comment:
Also todo?
```suggestion
	finale() //qwertodo low priority: this isn't working right?
	return ..()
```

Reply:
I wonder if that will fix the issue, (apart from needing to do it), applied